.gitignore

```diff
@@ -39,3 +40,4 @@ callgrind.out.*
 tests/autotests/autotests
 .ycm_extra_conf.py*
 *.stash
+
```

Comment:
> You can have `vim` also on mac

That's what I ended up using in the end about 5 hours later... but some changes were already done in other editors. Qt Creators is awful too on the Mac. All the GUI ones seriously slow down the VNC session. e.g. click... whistle for up to 30 seconds sometimes... then it renders the context menu. Lost a few hairs tonight. ;)